package.json

```diff
@@ -22,47 +22,47 @@
     "content": "CC­BY­NC­SA 4.0"
   },
   "devDependencies": {
-    "active-model-adapter": "2.1.1",
-    "broccoli-asset-rev": "2.5.0",
-    "ember-ajax": "2.5.3",
-    "ember-cli": "2.10.0",
-    "ember-cli-app-version": "2.0.1",
-    "ember-cli-autoprefixer": "0.6.0",
-    "ember-cli-babel": "5.2.1",
-    "ember-cli-dependency-checker": "1.3.0",
-    "ember-cli-flash": "1.4.2",
-    "ember-cli-htmlbars": "1.1.1",
-    "ember-cli-htmlbars-inline-precompile": "0.3.6",
-    "ember-cli-inject-live-reload": "1.4.1",
-    "ember-cli-jshint": "2.0.1",
-    "ember-cli-moment-shim": "3.3.1",
-    "ember-cli-qunit": "3.0.4",
-    "ember-cli-release": "0.2.9",
-    "ember-cli-sass": "6.2.0",
-    "ember-cli-sri": "2.1.1",
-    "ember-cli-test-loader": "1.1.0",
-    "ember-cli-uglify": "1.2.0",
-    "ember-data": "2.10.0",
-    "ember-disable-proxy-controllers": "1.0.1",
-    "ember-export-application-global": "1.1.1",
-    "ember-load-initializers": "0.6.3",
-    "ember-local-storage": "1.3.7",
-    "ember-moment": "7.3.1",
-    "ember-paper": "1.0.0-alpha.20",
-    "ember-resolver": "2.1.0",
-    "grunt": "1.0.1",
-    "grunt-concurrent": "2.3.1",
-    "grunt-contrib-jshint": "1.1.0",
-    "grunt-jscs": "3.0.1",
-    "grunt-jsonlint": "1.1.0",
-    "load-grunt-config": "0.19.2",
-    "loader.js": "4.1.0",
-    "time-grunt": "1.4.0"
+    "active-model-adapter": "^2.1.1",
```

Comment:
did you have to bump all of these?, I'd always recommend hardcoding these. 
https://www.theregister.co.uk/2016/03/23/npm_left_pad_chaos/